### PR TITLE
1.18.0

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/pypa/hatch/tree/hatchling-v1.17.1/backend)

### CODE REVIEW

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,11 +22,12 @@ requirements:
   host:
     - pip
     - python
-    # bootstrapping
-    - packaging >=21.3
-    - pathspec >=0.10.1
-    - pluggy >=1.0.0
-    - tomli >=2.0
+    - wheel
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:25]: `wheel` shouldn't be required. `hatchling` uses its own code to package itself, see https://github.com/pypa/hatch/blob/master/backend/pyproject.toml#L3-L4.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set name = "hatchling" %}
-{% set version = "1.12.2" %}
+{% set version = "1.17.1" %}
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:2]: We might want to update to 1.18.0 since it's the latest version.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -22,11 +22,12 @@ requirements:
...
-    - packaging >=21.3
-    - pathspec >=0.10.1
-    - pluggy >=1.0.0
-    - tomli >=2.0
+    - wheel
+    - packaging 21.3
+    - pathspec 0.10.3
+    - pluggy 1.0.0
+    - tomli 1.2.2
+    - trove-classifiers 2023.3.9
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:30]: `editables` is missing from the list of requirements, see https://github.com/pypa/hatch/blob/master/backend/pyproject.toml#L37.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -22,11 +22,12 @@ requirements:
...
     - pip
     - python
-    # bootstrapping
-    - packaging >=21.3
-    - pathspec >=0.10.1
-    - pluggy >=1.0.0
-    - tomli >=2.0
+    - wheel
+    - packaging 21.3
+    - pathspec 0.10.3
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:27]: In theory, it requires >=0.10.1

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 4 above)
Comment (SC426) [reply recipe/meta.yaml:27]: Just keeping the linter satisfied

Comment (SC426) [reply recipe/meta.yaml:27]: 0.10.3 is the closest we have

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**